internal/wire/parse.go

```diff
@@ -372,9 +374,41 @@ func load(ctx context.Context, wd string, env []string, patterns []string) ([]*p
 	if len(errs) > 0 {
 		return nil, errs
 	}
+	// Look for and examine the imported wire library.
+LOOP:
+	for _, pkg := range pkgs {
+		for path, p := range pkg.Imports {
+			if isWireImport(path) {
+				loadWireLib(p)
+				break LOOP
+			}
+		}
+	}
 	return pkgs, nil
 }
 
+// loadWireLib checks the wire library version to decide the set of behavior of
+// the generator.
+func loadWireLib(pkg *packages.Package) {
+	for _, f := range pkg.Syntax {
```

Comment:
An AST walk works to find the constant, but I wonder if we could use the type-checking info (`pkg.Types`) to do a scope lookup and get it more directly (`pkg.Types.Scope().Lookup("bindToUsePointer")`).